python/py-terminaltables/Portfile

```diff
@@ -24,7 +27,13 @@ checksums           rmd160  9d02523f89d32c890ae9c40212155e221584e279 \
                     size    12264
 
 if {${name} ne ${subport}} {
-    depends_build-append    port:py${python.version}-setuptools
-
-    livecheck.type  none
+    # this patch is from git-format-patch,
+    # which requires flag `-p1`, instead of
+    # the macports portpatch default of `-p0`.
```

Comment:
these comments are all redundant; it's clear from the patchfile that you need the `-p1` ; please remove

Reply:
Done